src/storage.js

```diff
@@ -0,0 +1,15 @@
+export default class LocalStorage {
+  // retrieve to-do list
+  static getList() {
+    let toDoList = JSON.parse(localStorage.getItem('toDoList'));
+    if (!toDoList) {
+      toDoList = [];
+    }
+    return toDoList;
+  }
+
+  // save list to local storage
+  static save(list) {
+    localStorage.setItem('toDoList', JSON.stringify(list));
+  }
+}
```

Comment:
- [ ] Make sure that the data are being stored in the localStorage.

- The updated array of items is stored in local storage, so the user gets the correct list values after the page reloads.

Reply:
I am unsure what you mean?